### ISSUE TITLE
Latest script versions not working for me

### ISSUE THREAD

**Issue by azmau** [Purfview/IMDb-Scout-Mod#126]
Hi,

For a few weeks now the script no longer works for me. I kept waiting for an update and it seems like it was released, I have it but still does not work for me.
I've tried disabling the script, refreshing the page, enabling back the script and refreshing again the page without success.
I've tried restarting Chrome, nothing!
I'm up to date with both Chrome and Tampermonkey. What else can I do? What to check, what to reset?
Please help!
![image](https://user-images.githubusercontent.com/23192344/185207731-997698ec-90b8-4c9f-b493-a00b892dae0f.png)
![image](https://user-images.githubusercontent.com/23192344/185208807-2622a6c2-2ff9-437d-99f5-fd25e6589e9e.png)

**Comment by Purfview** [comment]
What is that addon with IMDb icon I see there? Maybe it interferes somehow.
Tampermonkey's icon doesn't show any scrip matched your URL there.
And there is "www" missing from your URL, it should be www.imdb.com/title/tt1488181

**Comment by azmau** [comment]
That addon is only helping me quickly search for highlighted text on IMDb via right-click. I deactivated it and still no improvement.
Indeed, the Tampermonkey icon doesn't show any match, I also noticed that but don't know what to check more, it seems to me like everything should be working.
The "www" is not really missing, it's just hidden by Chrome. Also clicking the link in your comment I get the same behavior.
![image](https://user-images.githubusercontent.com/23192344/185244765-ce56313a-e9b7-4925-8aa6-afd9f83c77c9.png)

**Comment by Purfview** [comment]
Check the browser's console maybe there will be some info.
Try to disable all addons but Monkey.
Use other scripts manager like Violentmonkey.

**Comment by azmau** [comment]
Thanks a lot @Purfview , but I think I won't bother you anymore. I had one other script that was for Twitter and after activation I noticed that it is not matching either.
So its not your script, more likely Tampermonkey is to blame. Even though I tried again on incognito mode (with all extensions not allowed to work in incognito except TM like you suggested) and it just won't work. I guess I'll wait a bit more for a TM update or try in the end your suggestion for Violentmonkey.
Hope to make it work asap, I really enjoyed your script. Keep up the good work!
